metahtml/timestamp.py

```diff
@@ -379,6 +379,27 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'time.com',                       '//div[contains(@class,"published-date")]' ),
         ( 'townhall.com',                   '//div[@class="contributor pull-left"][contains(.,"Posted:")]/text()'),
         ( 'voxeurop.eu',                    '//div[contains(@class,"publish_date_time")]' ),
+        ( 'today.it',                       '//time[@class="datestamp"]' ),
+        ( 'shine.cn',                       '//meta[@name="ArticlePublishDate"]/@content'),
+        ( 'nbcnews.com',                    '//span[@class="byline___3V_nz"]'),
+        ( 'nbcnews.com',                    '//meta[@itemprop="datePublished"]/@content'),
+        ( 'onenewspage.co.uk',              '//span[@class="lhpt"]'),
+        ( 'abc.net.au',                     '//meta[@name="DCTERMS.issued"]/@content'),
+        ( 'aljazeera.com',                  '//div[@class="article-duration"]/time/@datetime'),
+        ( 'buzzfeednews.com',               '//p[@class="news-article-header__timestamps-posted"]'),
+        ( 'thanhnien.vn',                   '//meta[@itemprop="datePublished"]/@content'),
+        ( 'rt.com',                         '//meta[@name="published_time_telegram"]/@content'),
+        ( 'newindianexpress.com',           '(//p[@class="ArticlePublish margin-bottom-10"]/span)[1]'),
+        ( 'freepressjournal.in',            '//div[@class="story-meta-info-m__date-time__1IiJn"]/time/@datetime'),
+        ( 'news.com.au',                    '//meta[@itemprop="datePublished"]/@content'),
+        ( 'actu.orange.fr',                 '//div[@class="player-head"]/meta[@itemprop="uploadDate"]/@content'),  # TODO
+        #( 'tienphong.vn',                   '//meta[@class="cns-date"]/@content' ),
```

Comment:
You shouldn't be leaving commented out lines like this in source code that gets committed.